pkgs/applications/networking/firewalk/default.nix

```
@@ -0,0 +1,25 @@
+let
+  nixpkgs = import <nixpkgs> {  };
+
+in
+  with nixpkgs;
```

Please mention the inputs:

```suggestion
{ lib
, stdenv
, fetchurl
, libnet
...
}:
```

---

Thanks, Done!